### PR TITLE
Bug/change repeat meds path

### PR DESCRIPTION
**Pull request by natasafrgk**

### CODE REVIEW

File: lambdas/data_mapping/lib/medications_mapping.py [hunk 1]
```diff
@@ -36,23 +39,25 @@ def map_medications(statements: list[dict], requests: list[dict], medications: l
...
 
-    medication_type = get_simple_mapping("type", medication, TYPE_PATH)
+    medication_type = get_multiple_paths_mapping("type", medication, MED_TYPE_PATHS)
 
     keys = medication.keys()
 
     find_missing_group(keys)  # type: ignore
 
     mapping = {
-        "type": medication_type,
```
Comment (ed-bright) [inline lambdas/data_mapping/lib/medications_mapping.py:46]: Why this change?

File: lambdas/data_mapping/lib/medications_mapping.py [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
         for medication in grouped_meds
-        if get_simple_mapping("type", medication, TYPE_PATH).lower() == "acute"
+        if "Acute" == get_multiple_paths_mapping("type", medication, MED_TYPE_PATHS)
     ]
 
     repeat_meds = [
         map_medications_resource(medication)
         for medication in grouped_meds
-        if get_simple_mapping("type", medication, TYPE_PATH).lower() == "repeat"
+        if "Repeat" == get_multiple_paths_mapping("type", medication, MED_TYPE_PATHS)
```
Comment (ed-bright) [inline lambdas/data_mapping/lib/medications_mapping.py:33]: These seem the wrong way round - should be if get_multiple == "Repeat"

File: lambdas/data_mapping/lib/medications_mapping.py [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
+
+def map_medications(statements: list[dict], requests: list[any], medications: list[dict]) -> dict:  # type: ignore
     """Orchestration function to co-ordinate-field-mapping"""
     grouped_meds = group_medications(statements, requests, medications)
 
     acute_meds = [
         map_medications_resource(medication)
         for medication in grouped_meds
-        if get_simple_mapping("type", medication, TYPE_PATH).lower() == "acute"
+        if "Acute" == get_multiple_paths_mapping("type", medication, MED_TYPE_PATHS)
```
Comment (ed-bright) [inline lambdas/data_mapping/lib/medications_mapping.py:27]: Same her

File: lambdas/orchestration/lib/invoke_lambda.py [hunk 4]
```diff
@@ -17,7 +17,7 @@ def get_payload_response(response):
     body = json.loads(payload.get("body", "{}"))
     status_code = payload.get("statusCode")
     LOGGER.audit_dict["durations"] = {
-        **LOGGER.audit_dict.get("durations", {}),
+        **LOGGER.audit_dict.get("durations", {}),  # type: ignore
```
Comment (ed-bright) [inline lambdas/orchestration/lib/invoke_lambda.py:20]: Why is this neeed?

**Review by ed-bright — CHANGES_REQUESTED** [review]

File: lambdas/data_mapping/lib/medications_mapping.py (hunk 1 above)
Comment (natasafrgk) [reply lambdas/data_mapping/lib/medications_mapping.py:46]: Some of the tests were failing for it when it was just an empty string. Possibly breadcrumbs from various attempts of changing the def get_multiple_paths and trying to make it work. I tried it now as `"type": medication_type` and it worked, so I will revert.

File: lambdas/data_mapping/lib/medications_mapping.py (hunk 2 above)
Comment (natasafrgk) [reply lambdas/data_mapping/lib/medications_mapping.py:33]: yes, you are right here. I will revert.

File: lambdas/orchestration/lib/invoke_lambda.py (hunk 4 above)
Comment (natasafrgk) [reply lambdas/orchestration/lib/invoke_lambda.py:20]: It was giving me : Expected mapping for dictionary unpack operator

It doesn't affect anything, so I will revert.

**Review by ed-bright — APPROVED** [review]
